### DISCUSSION TITLE
[Feature] Github: A button to navigate to more useful page (e.g. profile) from eradicated hompage

### DISCUSSION THREAD

**Discussion by krishoang222** [jordwest/news-feed-eradicator#404] [Ideas]
## Issue
Navigating to the GitHub profile page (which I personally often use) from the eradicated homapage is required some steps:
1. Click the profile picture on the top right corner
2. Click "Your profile"

## Solution
I think we can provide an easier and more helpful way for users to do it:
- (marker 1) Set new URL for GitHub homepage button (currently point: https://github.com which has been eradicated already) to profile URL.
- (marker 2) Create a nav button below the quote, but this way make the design look different from other New Eradicator's pages (e.g. Twitter, LinkedIn)
- Bonus: we can let users decide which page they want to set this shortcut

## Screenshot
![image](https://github.com/jordwest/news-feed-eradicator/assets/74447462/4746e77c-10ed-4d5d-b7f1-343ba0196c7b)

**Comment by jordwest** [comment]
Check out https://github.com/jordwest/news-feed-eradicator/discussions/364 which looks like it would solve this
